File: Java/Program/Product.java
/*
Saya Muhamad Tio Ariyanto [2201718] mengerjakan soal Latihan Praktikum 2
dalam mata kuliah DPBO untuk keberkahanNya saya tidak melakukan kecurangan
seperti yang telah dispesifikasikan. Aamiin
*/

public class Product {

    private String idProduct;
    private String name;
    private String brand;
    private String price;

    public Product() {
        this.idProduct = "";
        this.name = "";
        this.brand = "";
        this.price = "";
    }

    public Product(String idProduct, String name, String brand, String price) {
        this.idProduct = idProduct;
        this.name = name;
        this.brand = brand;
        this.price = price;
    }

    public String getIdProduct() {
        return this.idProduct;
    }

    public void setIdProduct(String idProduct) {
        this.idProduct = idProduct;
    }

    public String getName() {
        return this.name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getBrand() {
        return this.brand;
    }

    public void setBrand(String brand) {
        this.brand = brand;
    }

    public String getPrice() {
        return this.price;
    }

    public void setPrice(String price) {
        this.price = price;
    }
}

File: Python/Program/Product.py
# Saya Muhamad Tio Ariyanto [2201718] mengerjakan soal Latihan Praktikum 2
# dalam mata kuliah DPBO untuk keberkahanNya saya tidak melakukan kecurangan
# seperti yang telah dispesifikasikan. Aamiin

class Product:
    # atribut
    __idProduct = ""
    __name = ""
    __brand = ""
    __price = ""
    
    # constructor
    def __init__(self, idProduct, name, brand, price):
        self.__idProduct = idProduct
        self.__name = name
        self.__brand = brand
        self.__price = price

    # setter
    def set_idProduct(self, idProduct):
        self.__idProduct = idProduct

    def set_name(self, name):
        self.__name = name

    def set_brand(self, brand):
        self.__brand = brand

    def set_price(self, price):
        self.__price = price

    # getter
    def get_idProduct(self):
        return self.__idProduct

    def get_name(self):
        return self.__name

    def get_brand(self):
        return self.__brand

    def get_price(self):
        return self.__price


